src/components/DownloadCards/DownloadCards.scss

```diff
@@ -1,11 +1,20 @@
+.download-cards-div {
```

Comment:
I suggest to pick more meaningful class name here. Maybe something like `download-cards-wrap` or `download-cards-group`

Reply:
@rodion-arr I have changed the class name.